services/user/AdminSys/ui/src/pages/dashboard-page.tsx

```diff
@@ -1,6 +1,11 @@
-const DASHBOARDS_PREFIX =
-    window.location.hostname === "localhost"
-        ? "http://localhost:8091/grafana"
+function isLocalhost() {
+    const hostname = window.location.hostname;
+    return hostname.includes('localhost') || hostname.includes('127.0.0.1');
```

Comment:
I would probably restrict the match to require any leading or trailing components to be separated by a `.`.  Also, one more way to spell localhost is: `[::1]`